app/models/cloud_subnet_network_port.rb

```diff
@@ -0,0 +1,6 @@
+class CloudSubnetNetworkPort < ApplicationRecord
+  self.table_name = "cloud_subnets_network_ports"
+
+  belongs_to :cloud_subnet
+  belongs_to :network_port
+end
```

Comment:
Is this model necessary at all?  I think it can be dropped unless I'm missing something.